src/openloco/windows/LandscapeGeneration.cpp

```diff
@@ -28,29 +28,7 @@ namespace openloco::ui::windows::LandscapeGeneration
 
     static loco_global<uint8_t, 0x00526247> industryFlags;
 
-    static loco_global<uint16_t, 0x009C8716> scenarioStartYear;
```

Comment:
References to these loco_globals have not yet been replaced with a reference to one of the SV5 structs, correct?